### ISSUE TITLE
dials.scale: with multi-wavelength data should merge each wavelength separately (by default)

### ISSUE THREAD

**Issue by graeme-winter** [dials/dials#2210]

**Comment by graeme-winter** [comment]
Interestingly `dials.export` on the output does _precisely_ what I would want i.e. separates out the wavelengths into different CCP4 / MTZ `dataset`s

**Comment by graeme-winter** [comment]
`dials.merge` takes a slightly different (but legitimate) approach to producing the MTZ output but still only produces one set of statistics.

**Comment by jbeilstenedmands** [comment]
I think `dials.merge` produces two sets of statistics, but not both at the bottom of the log?

**Comment by graeme-winter** [comment]
> I think `dials.merge` produces two sets of statistics, but not both at the bottom of the log?

Correct! But there was about a million lines of other stuff in between so I did not notice

**Comment by jbeilstenedmands** [comment]
Okay, having had a quick look, I think we can nicely unify the logging output from `dials.scale` and `dials.merge` for multi-wavelength cases, and also add a suitable wavelength tolerance parameter for `dials.scale` and `dials.export` to be consistent when running as a pipeline of programs. I guess it would also make sense to update the scaling html report to separate out multi-wavelength too, like the new-style dials.merge html report....

**Comment by graeme-winter** [comment]
> Okay, having had a quick look, I think we can nicely unify the logging output from `dials.scale` and `dials.merge` for multi-wavelength cases, and also add a suitable wavelength tolerance parameter for `dials.scale` and `dials.export` to be consistent when running as a pipeline of programs. I guess it would also make sense to update the scaling html report to separate out multi-wavelength too, like the new-style dials.merge html report....

Sounds very sensible thanks!

**Comment by dagewa** [comment]
@jbeilstenedmands I think this kind of thing was touched recently, right?
